virttest/libvirt_xml/vm_xml.py

```diff
@@ -2374,6 +2381,30 @@ def marshal_to_iothreadpins(tag, attr_dict, index, libvirtxml):
             return None
         return dict(attr_dict)
 
+    @staticmethod
+    def marshal_from_iothreadscheds(item, index, libvirtxml):
+        """
+        Convert a dict to iothreadsched tag and attributes.
+        """
+        del index
+        del libvirtxml
+        if not isinstance(item, dict):
+            raise xcepts.LibvirtXMLError("Expected a dictionary of host "
+                                         "attributes, not a %s"
+                                         % str(item))
```

Comment:
I think here should add  info of 'type(item)', but it looks the whole file uses the current way which is fine with me